tools/utils/download_blender.sh

```diff
@@ -64,20 +73,64 @@ function get_extractor() {
     echo "${extractor}"
 }
 
+function verify_download_integrity() {
+    local version=${1}
+    local target_filepath=${2}
+
+    if [ ! -f "${target_filepath}" ]; then
+        return 1
+    fi
+
+    echo "Found Blender ${ver} download. Verifying the integrity."
```

Comment:
I think `${ver}` should be `${version}`.

Reply:
good spot. There is a global variable `ver` defined in the main part of the script, so it still worked.